pages/account.tsx

```diff
@@ -0,0 +1,303 @@
+import { DeleteIcon, EditIcon } from "@chakra-ui/icons";
```

Comment:
Use heroicons (unless you think we should default to chakra then fallback on heroicons if chakra is missing an icon).

Reply:
That's what I've been doing but maybe you're right and I should just get rid of the Chakra icons dependency. They're so similar that I wouldn't be surprised if Chakra just ripped off Heroicons…